src/ttsnake.c

```diff
@@ -229,6 +234,16 @@ void draw (scene_t* scene, int number)
   if (number == GAME_OVER) { /* if displayed scene is the game over scene */
     /* we must show the final score */
     mvwprintw(main_window, NROWS*3/4, NCOLS/2-7, "Score: %d", block_count);
+    
+    /*Instructions to enter nickname*/
+    if(!entered_score) {
```

Comment:
```suggestion
    if(!entered_score) {
          mvwprintw(main_window, NROWS*3/4 + 1,NCOLS/5 - 2, "                                             ");
```

Adding this line hides the instruction to press 'r' to restart the game while the player is typing the nickname